integration/cluster/cluster.go

```diff
@@ -73,10 +73,17 @@ limits_config:
         attributes: [email]
 
 storage_config:
+  # Legacy config
   named_stores:
     filesystem:
       store-1:
         directory: {{.sharedDataPath}}/fs-store-1
+  # Thanos config
+  object_store:
+    named_stores:
+      filesystem:
+        store-1:
+          dir: {{.sharedDataPath}}/fs-store-1
```

Comment:
@ashwanthgoli Just to clarify, this required change is expected, right? Otherwise named stores won't work with Thanos.